src/shared/specifier.c

```diff
@@ -212,22 +213,10 @@ int specifier_architecture(char specifier, const void *data, const char *root, c
 }
 
 static int specifier_os_release_common(const char *field, const char *root, char **ret) {
```

Comment:
is there any point in having this wrapper after this change?

Reply:
Yeah, because the function interface is different. field,root,ret vs. root,field,ret.

Reply:
it's just used in the functions defined immediately below it - any reason not to call parse_os_release directly switching the parameters around?

Reply:
Indeed. Somehow I managed now to see that.